src/Components/Ignitor/src/PublicAPI.Shipped.txt

```diff
@@ -190,4 +190,4 @@ virtual Ignitor.Node.Parent.set -> void
 ~Ignitor.TextNode.TextNode(string text) -> void
 ~override Ignitor.RenderTreeFrame.ToString() -> string
 ~readonly Ignitor.ArrayRange<T>.Array -> T[]
-~readonly Ignitor.RenderTreeEdit.RemovedAttributeName -> string
+readonly Ignitor.RenderTreeEdit.RemovedAttributeName -> string?
```

Comment:
Hmm, we could turn off the PublicAPI analyzer on this project. We're not shipping this package.